source/getting-started.html.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: getting_started_layout
+---
+
+# Get started
+
+Welcome to GOV.UK.
+
+Our development environment is an Ubuntu virtual machine (VM). We aim to achieve [dev-prod parity](http://www.12factor.net/dev-prod-parity).
+
+Follow the steps on this page to get your GDS development environment running with a [VirtualBox](https://www.virtualbox.org/) VM, managed and configured by [Vagrant](http://vagrantup.com/).
+
+> You’ll need to be using a Mac to follow this guide.
+
+It's expected to take roughly a day to complete this guide, start to finish. There's a lot of things to download, and loads of installers need to do their thing.
+
+> You will need up to 100GB of free space on your hard-drive to run the whole of gov.uk.
+
+### Developing outside the VM
+
+You don't have to develop on the VM, but we strongly recommend it. If you have problems with the development VM you can always ask for help in the #govuk-developers Slack channel.
+
+### Where you should run commands
+
+Run mac$ commands in the shell on your Mac:
+
+    mac$ echo "Think Different"
+
+Run dev$ commands in the shell on the development VM:
+
+    dev$ echo "Linux for human beings"
+
+## 1. Get set up
+
+First, install:
+
+* [OSX GCC tools](https://github.com/kennethreitz/osx-gcc-installer)
+* [VirtualBox](https://www.virtualbox.org/)
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* vagrant-dns plugin: ` vagrant plugin install vagrant-dns `
+
+You’ll then need to:
+
+1. Set up a [GitHub](https://www.github.com) account.
+1. Get yourself added to the [alphagov organisation](https://github.com/alphagov). Your tech lead will be able to facilitate this.
+1. [Generate and register an SSH key pair](https://help.github.com/categories/56/articles) for your Mac for your GitHub account.
+1. Import the SSH key into your keychain. Once you’ve done this, it’ll be available to the VM you'll install in the next step.
+
+        mac$ /usr/bin/ssh-add -K yourkey
+
+1. Test that it all works by running `ssh -T git@github.com`
+
+## 2. Create a user account on our remote servers
+
+Follow the [Puppet repository instructions](https://github.com/alphagov/govuk-puppet/blob/master/docs/creating-a-user-account.md) to create an account.
```

Comment:
These instructions aren't very user-friendly. I suggest we write some ourselves for this section.